src/AuthContext.tsx

```diff
@@ -96,7 +102,10 @@ export const AuthProvider = ({ authConfig, children }: IAuthProvider) => {
   function handleTokenResponse(response: TTokenResponse) {
     setToken(response.access_token)
     setRefreshToken(response.refresh_token)
-    const tokenExpiresIn = config.tokenExpiresIn ?? response.expires_in ?? FALLBACK_EXPIRE_TIME
+    const tokenExpiresIn = config.tokenExpiresIn
```

Comment:
Looks good. Tested and seem to work as expected.
However, I think we should rearrange these setStates, so that you only decode the JWT once.